docs/quick-start-guide.adoc

```diff
@@ -33,10 +33,22 @@ WARNING: This guide assumes you already know how to deal with command line and k
                 "path": "/wd/hub"
             }
         }
+    },
+    "chrome": {
```

Comment:
Let's not include this to JSON because otherwise we need to do `docker pull` on the next step.